packages/calypso-e2e/src/lib/components/sidebar-component.ts

```diff
@@ -42,9 +40,10 @@ export class SidebarComponent {
 	 *
 	 * @param {string} item Plaintext representation of the top level heading.
 	 * @param {string} subitem Plaintext representation of the child level heading.
+	 * @param {number} _retries Number of retries in the case of navigation failure.
 	 * @returns {Promise<void>} No return value.
 	 */
-	async navigate( item: string, subitem?: string ): Promise< void > {
+	async navigate( item: string, subitem?: string, _retries = 3 ): Promise< void > {
```

Comment:
Complete nitpick here, but I think you can take off the `_` prefix here! I think that would match the surrounding style better. But that's take it or leave it! :)

Reply:
Removed in https://github.com/Automattic/wp-calypso/pull/55654/commits/aafc6117b196ad6c8a97444fee6fc36348d09230